lusSTR/scripts/marker.py

```diff
@@ -1453,10 +1453,13 @@ def convert(self):
     def custom_brack(self):
         if self.custom:
             sequence = self.custom_sequence
-            final_string = (
-                f"{collapse_repeats_by_length(sequence[:14], 4)} "
-                f"{collapse_repeats_by_length(sequence[14:], 4)}"
-            )
+            if len(sequence) > 14:
+                final_string = (
+                    f"{collapse_repeats_by_length(sequence[:14], 4)} "
+                    f"{collapse_repeats_by_length(sequence[14:], 4)}"
+                )
+            else:
+                final_string = collapse_repeats_by_length(sequence, 4)
```

Comment:
I'm curious about the significance of the value 14 here.

Reply:
It's ensuring the beginning of the bracketed form looks like this ```[GAAA]3 AG GAAG ...```.